packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -153,11 +161,9 @@ - (void)signIn:(GIDSignIn *)signIn
 #pragma mark - private methods
 
 - (void)respondWithAccount:(id)account error:(NSError *)error {
-  NSArray<FlutterResult> *requests = _accountRequests;
-  _accountRequests = [[NSMutableArray alloc] init];
-  for (FlutterResult accountRequest in requests) {
-    accountRequest(error != nil ? error.flutterError : account);
-  }
+  FlutterResult result = _accountRequest;
```

Comment:
This _looks_ dicey since it is neither a clone nor an apparent pointer reassignment however it works. Let me know if any concerns.